suite-common/wallet-utils/src/transactionUtils.ts

```diff
@@ -23,14 +23,15 @@ export const sortByBlockHeight = (a: WalletAccountTransaction, b: WalletAccountT
     return blockB - blockA;
 };
 
+/**
+ * Returns transactions for the active account that have been fetched so far.
+ * The list is not sorted here because it may contain null values as placeholders
+ * for transactions that have not been fetched yet. (This affects pagination.)
+ */
 export const getAccountTransactions = (
     accountKey: string,
     transactions: Record<string, WalletAccountTransaction[]>,
-) => {
-    const txs = transactions[accountKey] || [];
-    // clone array (read-only from reducer) and sort by blockHeight
-    return txs.slice(0).sort(sortByBlockHeight);
-};
+) => transactions[accountKey] || [];
```

Comment:
Perhaps you could add a null check for transactions here?

Reply:
As discussed, probably better to let it fail in an unexpected case.